win/pkgs/grpc.ps1

```diff
@@ -0,0 +1,70 @@
+#Requires -RunAsAdministrator
+
+Get-Content "$PSScriptRoot/utils/re-entry.ps1" -Raw | Invoke-Expression
+$ErrorActionPreference="Stop"
+
+. "$PSScriptRoot/env/mirror.ps1"
+# . "$PSScriptRoot/env/toolchain.ps1"
+
+pushd ${Env:TMP}
+
+$repo="${Env:GIT_MIRROR}/grpc/grpc.git"
+$proj="$($repo -replace '.*/','' -replace '.git$','')"
+$root="${Env:TMP}/$proj"
+
+rm -Force -Recurse -ErrorAction SilentlyContinue -WarningAction SilentlyContinue "$root"
+if (Test-Path "$root")
+{
+    Write-Output "Failed to remove `"$root`""
+    Exit 1
+}
+
+$latest_ver='v' + $($(git ls-remote --tags "$repo") -match '.*refs/tags/v[0-9\.]*$' -replace '.*refs/tags/v','' | Sort-Object {[Version]$_})[-1]
+git clone --depth 1 --single-branch -b "$latest_ver" "$repo"
+pushd "$root"
+
+mkdir build-win
+pushd build-win
+
+cmake                                                               `
+    -DBUILD_SHARED_LIBS=OFF                                         `
+    -DCMAKE_BUILD_TYPE=Release                                      `
+    -DgRPC_BUILD_TESTS=OFF                                          `
+    -DCMAKE_C_FLAGS="/DPROTOBUF_USE_DLLS /GL /MP /Zi"               `
+    -DCMAKE_CXX_FLAGS="/DPROTOBUF_USE_DLLS /EHsc /GL /MP /Zi"       `
+    -DCMAKE_EXE_LINKER_FLAGS="/DEBUG:FASTLINK /LTCG:incremental"    `
+    -DCMAKE_PDB_OUTPUT_DIRECTORY="${PWD}/pdb"                       `
+    -DCMAKE_SHARED_LINKER_FLAGS="/DEBUG:FASTLINK /LTCG:incremental" `
+    -DCMAKE_STATIC_LINKER_FLAGS="/LTCG:incremental"                 `
+    -DCMAKE_INSTALL_PREFIX="${Env:ProgramFiles}\grpc"               `
```

Comment:
Please don't.
You should respect the default place on Windows so that it's easier for other packages to find, unless:
1. You're install to a temp folder for packaging, which will be relocated later.
2. There's inconsistency across version (gtest for example, uses googletest, googletest-distribution, ...)
3. The name is ridiculous (e.g. Intel(R) MKL-DNN ...)

Reply:
I see. That's why there're ProgramFiles and ProgramFiles(x86) mixed in the repo, got it.